src/lib/Libattr/attr_node_func.c

```diff
@@ -291,8 +291,7 @@ str_to_vnode_state(char *vnstate)
  *
  * @return 	int
  * @retval	<0	an error encountered; value is negative of an error code
- * @retval	>=0 ok, encode happened and number of svrattrl created and linked in,
- *		     	or nothing to encode.
+ * @retval ==1 ok, encode succeeded and returning one item 
```

Comment:
Could you line things up a little better?

Reply:
sure will do